docs/ref/contrib/gis/forms-api.txt

```diff
@@ -164,18 +164,28 @@ Widget classes
         isn't suitable for production use since it offers no guaranteed uptime
         and runs on a slow server.
 
-    .. _tailored to your needs: http://docs.openlayers.org/library/deploying.html
+        Moreover, OpenLayers 2.x has been dropped in favor of OpenLayers 3. If
```

Comment:
possibly more clear: "Also, the widget nows uses OpenLayers 3 instead of OpenLayers 2."

I think the note about "If you extend..." is sufficient in the release notes.